skills/dff_voice_skill/scenario/response.py

```diff
@@ -0,0 +1,22 @@
+import logging
+import common.dff.integration.context as int_ctx
+
+from df_engine.core import Context, Actor
+
+
+logger = logging.getLogger(__name__)
+
+
+def caption(ctx: Context, actor: Actor, excluded_skills=None, *args, **kwargs) -> str:
+    cap = "ERROR"
+    if not ctx.validation:
+        cap = (
+            int_ctx.get_last_human_utterance(ctx, actor)
+            .get("annotations", {})
+            .get("voice_service", {})
+            .get("captions", "No cap")
```

Comment:
where is the `no cap` dealing?

Reply:
Elaborate please

Reply:
you may return ` f"Is there No cap in that audio?"` - but this is strange. Make some default response, if you have an audio attached but could not caption it -> some question like "I could not read your audio, attach another one"

Reply:
Changed so that if there is no caption, another message is sent